### PR TITLE
Docs: Improved search type table

### PR DESCRIPTION
**Pull request by vassudanagunta**
- fixed grammar
- removed redundant "query"
- split title and body into separate rows (since there are only two field, easier to read if they are separate. When Joplin adds more fields it will make more sense to combine them again)
- made it clear that the different types can be combined.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,10 +292,12 @@ Joplin implements the SQLite Full Text Search (FTS4) extension. It means the con
...
+Phrase | Add double quotes to return the notes that contain exactly this phrase. | `"shopping list"` - will return the notes that contain these **exact terms** next to each other and in this order. It will **not** return for example a note that contains "going shopping with my list".
 Prefix | Add a wildcard to return all the notes that contain a term with a specified prefix. | `swim*` - will return all the notes that contain eg. "swim", but also "swimming", "swimsuit", etc. IMPORTANT: The wildcard **can only be at the end** - it will be ignored at the beginning of a word (eg. `*swim`) and will be treated as a literal asterisk in the middle of a word (eg. `ast*rix`)
-Field restricted | Add either `title:` or `body:` before a note to restrict your search to just the title, or just the body. | `title:shopping`, `body:egg`
+Title | Prefix your query with `title:` to search note titles for matches. | `title:shopping`
+Body | Prefix your query with `body:` to search note content for matches. | `body:egg`
+*combination* | Combine the above types as needed. | `title:shop*`, `body:"shopping list"`
```
Comment (laurent22) [inline README.md:300]: Does combination actually work? Otherwise I like your changes but if you could confirm this that would be great.

Comment (vassudanagunta) [reply README.md:300]: Ok, I don't think it does. I'll revert that last change and resubmit.

Comment (vassudanagunta) [reply README.md]: not sure how a README change can result in build failure. Rebased onto head, still same. I assume the problem is upstream.

**Comment by laurent22** [comment]
Looks good, thanks for the update!

**Pull request merged**